Задача 3. Сравните 1 и 2 е измерения, предполагая, что 3го измерения через 30 минут не было. Есть
ли статистически значимые различия между измерениями давления?

In [1]:
import numpy as np 
import scipy.stats as stats
x1=[150, 160, 165, 145, 155]
x2=[140, 155, 150, 130, 135]
a=0.05
stats.wilcoxon(x1,x2)

WilcoxonResult(statistic=0.0, pvalue=0.0625)

pvalue=0.063 > a=0.05 ,  это означает, что статистических различий между измерениями давления не имеются (гипотеза Но не отвергается), препарат не оказал влияние через 10 минут после применения(значит его эффект проявляется полсе 30 мин наблюдения) :)